### ISSUE TITLE
Update Testimonial Readme

### ISSUE THREAD

**Issue by RavindraP04** [Clueless-Community/seamless-ui#1986]
Need to improve this component on `Testimonial/readme.md`

## Improvements required :
Update the Button readme as per newly designed Button components 

We have fully re-made whole `Testimonial` components as per our new design system, But the `Testimonial/readme.md` are still of old design system,
so now you have to copy paste the code of each component on play.tailwindcss.com and take screenshot of it, add that screenshot of the component in readme so that when someone clicks on that image, they will be redirected to the code of that particular component, All the images should be of equal aspect ratio like 16:9

**Comment by Priyamakeshwari** [comment]
I would like to work on this issue please assign me

**Comment by KrunalGupta02** [comment]
Hi i would like to work on it under jwoc

**Comment by KrunalGupta02** [comment]
@RavindraP04 photo are not working in many of files

**Comment by RavindraP04** [comment]
> @RavindraP04 photo are not working in many of files

@KrunalGupta02 Not Working means Not Loading ?
Where are you uploading the image ?

**Comment by KrunalGupta02** [comment]
Not loading 
I am not uploading any image i only paste the HTML file in the tailwind playground

**Comment by KrunalGupta02** [comment]
@RavindraP04 
![image](https://user-images.githubusercontent.com/101408619/223657648-e9aa3ec2-1358-4354-9d29-45044dd10a0e.png)

The problem I am facing is image is not loading and I how to center all the images I have tried the "width:100%" but it is looking good.
Kindly guide me on this.

**Comment by RavindraP04** [comment]
@KrunalGupta02 First thing is width 100% will not work here and that is why you have to take all the screenshots in equal ratio so that it perfectly fits the cells and also you have to upload the screenshot of the components on https://imgbb.com/ otherwise it won't render

**Comment by KrunalGupta02** [comment]
@RavindraP04
okk 👍🏻 
and for the image in the components which is not loading what can i do

**Comment by RavindraP04** [comment]
@KrunalGupta02 For each component in which image is not loading, Raise an issue separately and one by one only(because won't be able to assign more than one issue at a time) then, I will assign you in it under jwoc marking all of then as easy OR you can do raise a single issue for all the image issue and I'll mark it as medium.

**Comment by KrunalGupta02** [comment]
Ok, I will raise a single issue for each component.

**Comment by Laxmankohar** [comment]
@RavindraP04 , I would like to work on this issue under jwoc . Can you assingn it to me?

**Comment by prafulbbandre** [comment]
Hey @RavindraP04 i want to work on it.

**Comment by prafulbbandre** [comment]
what's the dead line for this issue.

**Comment by AkshayPratapSingh09** [comment]
Hey devs I  would like to work on this issue !  I think I would be able to pull this off easily!!

**Comment by RavindraP04** [comment]
> Hey devs I would like to work on this issue ! I think I would be able to pull this off easily!!

Assigned @AkshayPratapSingh09

**Comment by AkshayPratapSingh09** [comment]
@RavindraP04  Many image seems to be not working in this Testimonial it says access denied

**Comment by RavindraP04** [comment]
> @RavindraP04 Many image seems to be not working in this Testimonial it says access denied

Yes, That's what you have to update, If you see the changes in the commit section, the components have been updated and so is their file names, so you have to take screenshot of each component and attach it with its corresponding code, so that when users click on any image, it redirects to the code of that component

**Comment by Vivek-GuptaXCode** [comment]
hello,is this issue open to work on?
